ketchup/main.py

```diff
@@ -188,13 +174,17 @@ def run():
             episodes_per_day_alternative = 3
             episodes_per_day = episodes_per_day_alternative * per_week_limit
 
-            weeks_required_alternative = math.ceil(episodes / episodes_per_day)
+            total_days_required_alternative = math.ceil(
+                episodes / episodes_per_day_alternative)
+
+            weeks_required_alternative = total_days_required_alternative / (
+                per_week_limit)
 
             realistic_end_date = start_date + dt.timedelta(
                 weeks=weeks_required)
 
             realistic_end_date_alternative = start_date + dt.timedelta(
-                weeks=weeks_required_alternative)
+                days=total_days_required_alternative)
```

Comment:
Adjusted logic of the code to take in simplified version of the function for the website. We have the backup of the original file type to revert this in the future.